puppet/zulip/files/cron.d/soft-deactivate-users

```diff
@@ -0,0 +1,6 @@
+SHELL=/bin/bash
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+USER=zulip
+
+# Cron job to soft deactivate long term idle users weekly.
+0 5 * * 7 zulip /home/zulip/deployments/current/manage.py soft_activate_deactivate_users -d --inactive-for 30 > /dev/null
```

Comment:
We don't need it necessarily, I chose 28 days as default value for this reason only but then thought it might be worth specifying the argument there so that its more discoverable. I am just going to remove it now though.
Thanks for the `>/dev/null` advice. Though I am curious about the reason for doing so.